editor/plugins/script_editor_plugin.cpp

```diff
@@ -3869,6 +3882,7 @@ ScriptEditor::ScriptEditor() {
 	erase_tab_confirm->set_ok_button_text(TTR("Save"));
 	erase_tab_confirm->add_button(TTR("Discard"), DisplayServer::get_singleton()->get_swap_cancel_ok(), "discard");
 	erase_tab_confirm->connect("confirmed", callable_mp(this, &ScriptEditor::_close_current_tab).bind(true));
+	erase_tab_confirm->connect("cancelled", callable_mp(this, &ScriptEditor::_clear_close_queue));
```

Comment:
There is no `cancelled` signal, so `_clear_close_queue` never gets called.
This leads to issues with the queue holding too many entries.